vumi/transports/dmark/dmark_ussd.py

```diff
@@ -19,6 +19,10 @@ class DmarkUssdTransportConfig(HttpRpcTransport.CONFIG_CLASS):
         " expires.",
         default=600, static=True)
 
+    fix_to_addr = ConfigBool(
+        "Whether or not to ensure that the to_addr is always starting with a "
+        "* and ending with a #", default=False, static=True)
```

Comment:
Inclined to keep as is as all the config vars in there are currently marked as static and I'm using the `get_static_config()` convenience helper.